includes/widgets/text-editor.php

```diff
@@ -408,21 +408,22 @@ protected function register_controls() {
 	 * @access protected
 	 */
 	protected function render() {
-		$editor_content = $this->get_settings_for_display( 'editor' );
+		$is_edit_mode = Plugin::$instance->editor->is_edit_mode();
 
+		$editor_content = $this->get_settings_for_display( 'editor' );
 		$editor_content = $this->parse_text_editor( $editor_content );
 
-		if ( ! Plugin::$instance->experiments->is_feature_active( 'e_dom_optimization' ) ) {
+		if ( $is_edit_mode ) {
```

Comment:
Where do you check if the Experiment is active? (PHP & JS)

Reply:
I don't.
After thinking about it, I've realized that it doesn't matter, we want this div only in edit mode, and we don't care about the DOM experiment.

Reply:
I can't revert it, please roll it back. it's for BC.

Reply:
Why? Looks like it should work for BC anyway.

Reply:
If the experiment is inactive the div should be printed.

Reply:
Done.